.github/release_plan.md

```diff
@@ -26,6 +26,7 @@
     -   [ ] Bump the version number to the next monthly ("YYYY.MM.0") release in the `main` branch
         -   [ ] `package.json`
         -   [ ] `package-lock.json`
+    -   [ ] Merge the changes from release (Changelog, delete news, ThirdPartyNotices) into `main` branch
```

Comment:
Thanks to @DonJayamanne for noticing that I missed this. Updating the release_plan.